lib/api/skus.js

```diff
@@ -14,12 +17,64 @@ di.annotate(skusRouterFactory, new di.Inject(
         'Http.Services.RestApi',
         '_',
         'Promise',
-        'Protocol.TaskGraphRunner'
+        'Protocol.TaskGraphRunner',
+        'uuid',
+        'Http.Services.SkuPack',
+        'osTmpdir',
+        'rimraf'
     )
 );
 
-function skusRouterFactory (waterline, rest, _, Promise, taskGraphProtocol) {
+function skusRouterFactory (
+    waterline, 
+    rest, 
+    _, 
+    Promise, 
+    taskGraphProtocol, 
+    uuid, 
+    skuPack, 
+    tmp,
+    rimraf
+) {
     var router = express.Router();
+    var rimrafAsync = Promise.promisify(rimraf);
+
+    router.skuPackHandler = function(req,res,skuid) {
```

Comment:
Please add function documentation for your created new function follows [JSDoc](http://usejsdoc.org/) style.